src/ig/IGExporter.ts

```diff
@@ -197,7 +197,7 @@ export class IGExporter {
       // No need for the detailed log message since we already logged one in the package loader.
       logger.error(
         `Failed to add ${dependency.packageId} to ImplementationGuide instance because no ` +
-          'version was specified in your config.yaml.'
+          `version was specified in your ${path.basename(this.configPath)}.`
```

Comment:
Thanks for changing all the `config.yaml` to `${path.basename(this.configPath)}`.  That must have been pretty annoying.  You do `${path.basename(this.configPath)}` often enough, however, that it seems you might have benefited by just declaring a new `configName` property in the class, e.g.:
```ts
this.configName = path.basename(this.configPath);
```
and then `${this.configName}` everywhere.

Reply:
Yeah good point, added in https://github.com/FHIR/sushi/pull/586/commits/fedd542e656f51ffc3da5332ca16f1f8968e2660. Unfortunately that then dismissed your review and @jafeltra's review.